### PR TITLE
Fixes #636 : Added node `webpack` dep. to README.md

### LINKED ISSUE #636: Installation ERROR
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

image

**Expected behavior:**

A Clean Install.

**Screenshots of the issue:**

image

---

image

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Fixes #636 

## **Changes proposed in this pull request**
- **_Added a dependency named `webpack`_** :tada: 
- **_Added this fix to `loklak_search` documentation_** :+1: 

## **Screenshots (if appropriate)** 
image

### CODE REVIEW

**Comment by 0x48piraj** [comment]
**_Deploy Link : http://pr-637-fossasia-loklaksearch.surge.sh_**

Ping @jigyasa-grover @hemantjadon @mariobehling :heart:

File: README.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ After installing angular-cli we need to install our required node modules, so we
 ```sh
 $ npm install
 ```
+After installing node modules we need to install `webpack`, so we will do that by using the following command:
+```sh
+$ npm install webpack
```
Comment (singhpratyush) [inline README.md]: I am not sure if we require `webpack` as a project dependency. If so, why isn't is already there in the `package.json` file?

cc @hemantjadon @Achint08.

Comment (0x48piraj) [reply README.md]: I gave you the logs. I will check for this in `package.json`

**Comment by 0x48piraj** [comment]
Thanks Mario!

**Pull request merged**